apiary-metastore-events/sns-metastore-events/apiary-metastore-consumers/metastore-consumer-common/pom.xml

```diff
@@ -9,4 +9,39 @@
 
     <artifactId>metastore-consumer-common</artifactId>
 
+    <dependencies>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-metastore</artifactId>
```

Comment:
I thought we just agreed to have the scope at the child level?

Reply:
yeah - good catch. I had thought we were setting scope for just provided at the child originally but makes sense to be consistent with all scoping vs some scopes. Should all be accounted for now.